zerver/lib/generateData.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import itertools
+import json
+import random
+from typing import List, Dict, Any, Text, Optional
+from six.moves import range
+
+def loadConfig():
```

Comment:
(In the python-specific subset of zulip/zulip, but is not necessarily the case when the functions are cross-language?)